models/users.js

```diff
@@ -28,20 +28,22 @@ const addOrUpdate = async (userData, userId = null) => {
         })
       }
 
-      return { isNewUser, userId }
```

Comment:
This `isNewUser` is different than what we are storing in DataBase.
We use `isNewUser` in the return response to indicate if we Added a new user or Updated.
This is not related to profile completion.
Please revert to old changes.